wpilibc/src/main/native/cpp/ADXL362.cpp

```diff
@@ -24,13 +24,9 @@ static int kDataRegister = 0x0E;
 static int kFilterCtlRegister = 0x2C;
 static int kPowerCtlRegister = 0x2D;
 
-// static int kFilterCtl_Range2G = 0x00;
-// static int kFilterCtl_Range4G = 0x40;
-// static int kFilterCtl_Range8G = 0x80;
 static int kFilterCtl_ODR_100Hz = 0x03;
 
 static int kPowerCtl_UltraLowNoise = 0x20;
-// static int kPowerCtl_AutoSleep = 0x04;
```

Comment:
These are commented out to avoid unused warnings.  They're still valid constants.  IMO it's better to leave them in and commented out rather than having to find the docs again for them.